docker/Dockerfile.tcf-dev

```diff
@@ -111,17 +142,15 @@ RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.7.101.3.bin \
 
 
 WORKDIR /tmp
+ENV OPENSSL_VER=1.1.1d
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER /tmp/openssl-$OPENSSL_VER/
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER.tar.gz /tmp/
+
 # Build ("Untrusted") OpenSSL, then build ("trusted") SGX OpenSSL
```

Comment:
You could probably modify above comment as we are not building openSSL here ?

Reply:
ok